tool_statistics/statistics.py

```diff
@@ -99,11 +99,13 @@ def __init__(
 
     def run(self):
         grid_admin = str(self.result.parent().path.with_suffix('.h5'))
+        grid_admin_gpkg = str(self.result.parent().path.with_suffix('.gpkg'))
```

Comment:
Does this assume that the gridadmin.gpkg is always in the same directory as gridadmin.h5. Because that is not necessarily the case always

Reply:
it grabs the griadmin.gpkg file from the parent directory of the threedi result file. For example in here:

`.../threedi_results_analysis/{name_of_the_simulation}/gridadmin.gpkg`